### DISCUSSION TITLE
Option to toggle Custom Animation Rate being tied to Frame Rate

### DISCUSSION THREAD

**Discussion by Chickenfriedtoaster** [CapsAdmin/pac3#951] [Ideas]
Note, this is in reference to the Custom Animation part in advanced

When ever making custom animations that are either fast, or just have a lot of smaller frames (as the case with custom animations made and ported in blender) would slow down quite a lot whenever client framerate goes below a certain amount

Now while I'm not 100% sure if this is at all possible, though I do have a feeling it is, since ftime/frametime are a thing in proxies, however, I haven't gotten them to force the playing rate to behave.

I would imagine that this feature would have to also include a "Intended Animation Framerate" which would likely help with dealing with animation rate.

>Edited thrice, because Zeni keeps calling me a clown.

**Comment by CapsAdmin** [comment]
Yeah this is more like a bug, it shouldn't be dependent on framerate. I might be able to fix it without breaking existing animations.
